tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>The spacing between bullets is inconsistent. Strange...</p><p> </p><p>The PDSI link should point to <a href="https://developers.google.com/earth-engine/datasets/catalog/IDAHO_EPSCOR_PDSI" target="_blank">https://developers.google.com/earth-engine/datasets/catalog/IDAHO_EPSCOR_PDSI</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done.</p>